src/linear_algebra/affine_space/combination.lean

```diff
@@ -352,6 +352,22 @@ begin
   exact eq_weighted_vsub_of_point_subset_iff_eq_weighted_vsub_of_point_subtype
 end
 
+variables {k V}
+
+/-- Affine maps commute with affine combinations. -/
+lemma affine_map_combination {V₂ P₂ : Type*} [add_comm_group V₂] [module k V₂] [affine_space V₂ P₂]
```

Comment:
Shouldn't this be called `map_affine_combination`? If so, idem for the PR title.

Reply:
Yep, agreed!